### PR TITLE
Dockerfile definition

### PR DESCRIPTION
**Pull request by fredmaggiowski**
Hi, I wanted to contribute to the repository by adding a Dockerfile, useful for running the application inside a containerized environment or Kubernetes.

Since I wanted to use this service in a Mia-Platform project I also provided a bit of documentation on how to integrate the service once it will be available in the Marketplace.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+RUN npm run build
+
+RUN echo "commit: $COMMIT_SHA" >> ./commit.sha
+
+########################################################################################################################
+
+FROM node:14-alpine
+
+ENV NODE_ENV=production
```
Comment (enricop89) [inline Dockerfile:21]: can you add the required env variables please? 

`ENV NODE_ENV=production VIDEO_API_API_KEY=$VIDEO_API_API_KEY VIDEO_API_API_SECRET=$VIDEO_API_API_SECRET`

File: README.md [hunk 2]
```diff
@@ -110,3 +110,11 @@ It's the main component of the application. It contains the ToolBar and the netw
 ## Deploy to Heroku
 
 [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/nexmo-se/video-api-multiparty-toolkit-sample-app.git)
+
```
Comment (enricop89) [inline README.md:113]: I prefer to not add specific deployment instructions on the README file of this repo as its general purpose. Could you remove it, please?

**Comment by enricop89** [comment]
Added a few comments

File: README.md (hunk 2 above)
Comment (fredmaggiowski) [reply README.md:113]: ok no problem

**Comment by fredmaggiowski** [comment]
Hi @enricop89 do you think this one can be merged?

**Pull request merged**